docs/features/software-templates/input-examples.md

```diff
@@ -0,0 +1,209 @@
+---
+id: input-examples
+title: Built-in examples inputs
+description: Some examples to use in your template
+---
+
+All the examples on this page you can test using _create/edit_ from your Backstage installation.
+
+It is important to remember that all examples are based on [react-jsonschema-form](https://rjsf-team.github.io/react-jsonschema-form/).
+
+## Simple text input
+
+### Simple input with basic validations
+
+```yaml
+parameters:
+  - title: Fill in some steps
+    properties:
+      name:
+        title: Simple text input
+        type: string
+        description: Description about input
+        maxLength: 8
+        pattern: '^([a-zA-Z][a-zA-Z0-9]*)(-[a-zA-Z0-9]+)*$'
+        ui:autofocus: true
+        ui:help: 'Hint: additional description...'
+```
+
+### Simple secret input
+
+```yaml
+parameters:
+  - title: Fill in some steps
+    properties:
+      secretInput:
+        title: Input secret
+        type: string
+        description: Super secret description hint
+        minLength: 6
+        ui:widget: 'password'
+```
+
+### Multi line text input
+
+```yaml
+parameters:
+  - title: Fill in some steps
+    properties:
+      multiline:
+        title: Text area input
+        type: string
+        description: Insira o valor do input que gostraia
+        ui:widget: 'textarea'
+        ui:options:
+          rows: 10
+        ui:help: 'Hint: Make it strong!'
+        ui:placeholder: |
+          apiVersion: backstage.io/v1alpha1
+            kind: Component
+            metadata:
+              name: backstage
+            spec:
+              type: library
+              owner: CNCF
+              lifecycle: experimental
+```
+
+## Arrays options
+
+### Array with custom titles
+
+```yaml
+parameters:
+  - title: Fill in some steps
+    properties:
+      volume_type:
+        title: Volume Type
+        type: string
+        description: The volume type to be used.
+        default: 'gp2'
+        enum:
+          - 'gp2'
```

Comment:
You can drop the `'` from the beginning and end of pretty much all of these too I think and the other references in other files in yaml as quotes are not required.

Reply:
Nice its true ;)

Reply:
I will keep only in cases that have space